server/controllers/admin.py

```diff
@@ -440,7 +467,7 @@ def templates(cid, aid):
         if files:
             templates = {}
             for template in files:
-                templates[template.filename] = str(template.read(), 'latin1')
+                templates[template.filename] = str(template.read(), 'utf-8')
```

Comment:
^ that's specifically for web uploads. Keeping templates to something that's just utf-8 text should be fine.